Description: Lazarus FTBFS on mips and mipsel with:
 Error: Architecture mips is not available for elf format
 lazbuild.lpr(1871) Error: (9029) Error while compiling resources
 It seems like those archs are just missing to be defined
Author: Peter Green <plugwash@debian.org> and Paul Gevers <elbrus@debian.org>

Index: fpc/fpcsrc/utils/fpcres/target.pas
===================================================================
--- fpc.orig/fpcsrc/utils/fpcres/target.pas
+++ fpc/fpcsrc/utils/fpcres/target.pas
@@ -100,7 +100,7 @@ var
     (name : 'elf';      ext : '.or';     machines : [mti386,mtx86_64,mtppc,
                                                      mtppc64,mtarm,mtarmeb,
                                                      mtm68k,mtsparc,mtalpha,
-                                                     mtaarch64]),
+                                                     mtaarch64,mtmips,mtmipsel]),
     (name : 'coff';     ext : '.o';      machines : [mti386,mtx86_64,mtarm,
                                                      mtppc,mtppc64]),
     (name : 'xcoff';    ext : '.o';      machines : [mtppc{,mtppc64}]),
